android/app/src/main/java/com/microsoft/codepush/common/core/CodePushListeners.java

```diff
@@ -0,0 +1,37 @@
+package com.microsoft.codepush.common.core;
+
+import com.microsoft.codepush.common.interfaces.CodePushBinaryVersionMismatchListener;
+import com.microsoft.codepush.common.interfaces.CodePushDownloadProgressListener;
+import com.microsoft.codepush.common.interfaces.CodePushSyncStatusListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Incapsulates listeners that {@link CodePushBaseCore} is using.
+ */
+public class CodePushListeners {
+    /**
```

Comment:
missing space before comment?

Reply:
@max-mironov Yeah, I'll fix that, thanks Max.